commit 8a31486f47de0aa4dc5a7c5bf75892e691a73eab
Author: Conn <connogriofa@gmail.com>
Date:   Mon Jul 17 03:25:43 2017 +0100

    config: enhance DualShock3 controller support
    
    Enable rumble support in Sony HID & HID battery strength.

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 612504cf2acc..ed4a46b49d68 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -904,6 +904,7 @@ CONFIG_SND_SOC_AK4554=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_WM8804_I2C=m
 CONFIG_SND_SIMPLE_CARD=m
+CONFIG_HID_BATTERY_STRENGTH=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=m
 CONFIG_HID_A4TECH=m
@@ -946,6 +947,7 @@ CONFIG_HID_PICOLCD=m
 CONFIG_HID_ROCCAT=m
 CONFIG_HID_SAMSUNG=m
 CONFIG_HID_SONY=m
+CONFIG_SONY_FF=y
 CONFIG_HID_SPEEDLINK=m
 CONFIG_HID_SUNPLUS=m
 CONFIG_HID_GREENASIA=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 5cb81c879eb0..3cd9dca829e6 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -898,6 +898,7 @@ CONFIG_SND_SOC_AK4554=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_WM8804_I2C=m
 CONFIG_SND_SIMPLE_CARD=m
+CONFIG_HID_BATTERY_STRENGTH=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=m
 CONFIG_HID_A4TECH=m
@@ -940,6 +941,7 @@ CONFIG_HID_PICOLCD=m
 CONFIG_HID_ROCCAT=m
 CONFIG_HID_SAMSUNG=m
 CONFIG_HID_SONY=m
+CONFIG_SONY_FF=y
 CONFIG_HID_SPEEDLINK=m
 CONFIG_HID_SUNPLUS=m
 CONFIG_HID_GREENASIA=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1201_8a31486f47de0aa4dc5a7c5bf75892e691a73eab.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1201_8a31486f47de0aa4dc5a7c5bf75892e691a73eab.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
